README.md

```diff
@@ -14,7 +14,8 @@ For the documentation, please see our [wiki](https://github.com/e-valuation/EvaP
 
 ## Installation
 
-The easiest setup using [Vagrant](https://www.vagrantup.com) and [VirtualBox](https://www.virtualbox.org) is shown here. For manual installation instructions and production deployment please see the [wiki page on installation](https://github.com/e-valuation/EvaP/wiki/Installation).
+The easiest setup using [Vagrant](https://www.vagrantup.com) and [VirtualBox](https://www.virtualbox.org) is shown here. 
+Some CPUs like Apples M1/M2 are not supported by VirtualBox, please jump down to the docker setup section. For manual installation instructions and production deployment please see the [wiki page on installation](https://github.com/e-valuation/EvaP/wiki/Installation).
```

Comment:
> please jump down to the docker setup section

I think we should just inline the docker-specific parts into this guide instead of having a separate docker section. In the end, it should just be "additionally, run `vagrant provision`". But let's focus on the other parts first.

Reply:
If we inline them, we move it from experimental to supported, so we have to support it.

Reply:
I'd be fine with that. Whether we say that we support it doesn't really make a difference. People want to use it, if its broken they (rightfully) complain, so I'd just try to keep the readme simple.